### PR TITLE
explicit mention where to find model configurations

### PR DESCRIPTION
**Pull request by yzhang123**
Signed-off-by: Yang Zhang <yangzhang@nvidia.com>

### CODE REVIEW

File: docs/sources/source/nlp/bert_pretraining.rst [hunk 1]
```diff
@@ -7,7 +9,7 @@ Make sure you have ``nemo`` and ``nemo_nlp`` installed before starting this tuto
 The code used in this tutorial can be found at ``examples/nlp/language_modeling/bert_pretraining.py``.
 
 .. tip::
-    Pretrained BERT models can be found at 
+    Pretrained BERT models and model configuration files can be found at 
     `https://ngc.nvidia.com/catalog/models/nvidia:bertlargeuncasedfornemo <https://ngc.nvidia.com/catalog/models/nvidia:bertlargeuncasedfornemo>`__
```
Comment (ekmb) [inline docs/sources/source/nlp/bert_pretraining.rst]: should we add these links to the intro page maybe in a separate section?

Comment (yzhang123) [reply docs/sources/source/nlp/bert_pretraining.rst]: ok

File: docs/sources/source/nlp/bert_pretraining.rst [hunk 2]
```diff
@@ -6,11 +8,15 @@ Make sure you have ``nemo`` and ``nemo_nlp`` installed before starting this tuto
...
-    `https://ngc.nvidia.com/catalog/models/nvidia:bertbasecasedfornemo <https://ngc.nvidia.com/catalog/models/nvidia:bertbasecasedfornemo>`__
+.. _pretrained_models:
+
+Download pretrained models
+------------
+
+Pretrained BERT models and model configuration files can be found at 
+`https://ngc.nvidia.com/catalog/models/nvidia:bertlargeuncasedfornemo <https://ngc.nvidia.com/catalog/models/nvidia:bertlargeuncasedfornemo>`__
+`https://ngc.nvidia.com/catalog/models/nvidia:bertbaseuncasedfornemo <https://ngc.nvidia.com/catalog/models/nvidia:bertbaseuncasedfornemo>`__
+`https://ngc.nvidia.com/catalog/models/nvidia:bertbasecasedfornemo <https://ngc.nvidia.com/catalog/models/nvidia:bertbasecasedfornemo>`__
```
Comment (tkornuta-nvidia) [inline docs/sources/source/nlp/bert_pretraining.rst]: Can you please add few words of comments on those models? Think about user that doesn't have much experience, which model should he/she pick for his application?

File: docs/sources/source/nlp/question_answering.rst [hunk 3]
```diff
@@ -13,18 +15,20 @@ The pretrained back-bone models can be specified `--pretrained_model_name`.
...
 
-    For pretraining BERT in NeMo and pretrained model checkpoints go to `BERT pretraining <https://nvidia.github.io/NeMo/nlp/bert_pretraining.html>`__.
+Download pretrained models
+------------
+
+Finetuned SQuAD models and model configuration files can be found at 
+`https://ngc.nvidia.com/catalog/models/nvidia:bertlargeuncasedsquadv1 <https://ngc.nvidia.com/catalog/models/nvidia:bertlargeuncasedsquadv1>`__
+`https://ngc.nvidia.com/catalog/models/nvidia:bertlargeuncasedsquadv2 <https://ngc.nvidia.com/catalog/models/nvidia:bertlargeuncasedsquadv2>`__
+`https://ngc.nvidia.com/catalog/models/nvidia:bertbaseuncasedsquadv1 <https://ngc.nvidia.com/catalog/models/nvidia:bertbaseuncasedsquadv1>`__
+`https://ngc.nvidia.com/catalog/models/nvidia:bertbaseuncasedsquadv2 <https://ngc.nvidia.com/catalog/models/nvidia:bertbaseuncasedsquadv2>`__
```
Comment (tkornuta-nvidia) [inline docs/sources/source/nlp/question_answering.rst]: The same here, which one should I pick? And why?

**Pull request merged**